lynxmotion_al5d_description/Pose pose  # The new pose of the robot
---
